Makefile

```diff
@@ -107,6 +107,7 @@ install-external-extensions:
 install: install-server install-ui install-external-extensions ## Build and install
 	jupyter lab build
 	jupyter serverextension list
+	jupyter server extension list
```

Comment:
I believe that on master, which has migrated to "jupyter server" we can remove the live above using `serverextension`.

Reply:
we could remove it since we, elyra, doesn't use notebook server apps on master, but they are still supported by lab and "could" be installed still so I think leaving it in is worthwhile for clarity during development